qa/common/gen_qa_ragged_models.py

```diff
@@ -254,6 +280,108 @@ def create_plan_modelfile(models_dir, model_version, dtype):
     del builder
 
 
+def create_onnx_modelfile(models_dir, model_version, dtype):
+    # Create special identity model for batch input testing.
+    # Because the ragged input and batch input are one dimensional vector
+    # when passing to the model, the model must generate output with batch
+    # dimension so that Triton can scatter it to different responses along
+    # the batch dimension.
+    # 'BATCH_AND_SIZE_INPUT' is also used as a hint to generate output with
+    # batch dimension, 'BATCH_AND_SIZE_INPUT' must have shape [batch_size].
+    # Each output corresponds to the input with the same name, so if there
+    # are two requests, one has "RAGGED_INPUT" [2, 4] and the other has [1],
+    # since the input is ragged, the model sees the input as [2, 4, 1], and
+    # "BATCH_AND_SIZE_INPUT" will have shape [3]. Then the model output will
```

Comment:
`"BATCH_AND_SIZE_INPUT" will have shape [2]`, I think this typo also appears in other `create_xx_modelfile()`, do you mind to update those as well?